src/Conf.ml

```diff
@@ -990,6 +991,23 @@ module Formatting = struct
       (fun conf x -> {conf with module_item_spacing= x})
       (fun conf -> conf.module_item_spacing)
 
+  let nested_match =
+    let doc = "Style of let_and." in
+    let names = ["nested-match"] in
+    let all =
+      [ ( "wrap"
+        , `Wrap
+        , "$(b,wrap) classically wraps a match nested in another match \
+           case with parentheses and adds indentation." )
+      ; ( "align"
+        , `Align
+        , "$(b,align) does not indent and wrap a match if it is nested in \
```

Comment:
You are using the word `wrap` in the `align` case.
Why is the first case named `wrap` ? Wouldn't it be easier to understand if it was `default` ?

Reply:
The "not" applies to "indent" and to "wrap", but agreed, it can be rephrased.

Reply:
The "not" applies to "indent" and to "wrap", but agreed, it can be rephrased.